app/web/markdown/faq.md

```diff
@@ -51,18 +51,23 @@ If you want to help out in other ways, please get in touch and [join the team](/
 
 ### How do you plan on funding this? Doesn't it cost a lot of money?
 
-There are three main costs that Couchsurfing&#8482; needs to cover that we will need to replicate:
+The main cost that a couch surfing platform needs to cover are:
```

Comment:
Let's do it inline:

` The main cost that a couch surfing platform needs to cover is the cost of server hsoting and the third-party services required to operate the platform.`